draft-birkholz-scitt-software-use-cases.md

```diff
@@ -167,6 +167,35 @@ There is no standardized way to:
 * to check for the trustworthiness of a provider
 * to check the integrity of modifications or transformations done by a provider
 
+## Verifying Firmware Provenance for large and complex systems
+
+In contrast to operating systems or user space software components of a large and complex systems, firmware components are often already executed during boot-cycles before there is an opportunity to authenticate them.
```

Comment:
```suggestion
In contrast to operating systems or user space software components of large and complex systems, firmware components are often already executed during boot-cycles before there is an opportunity to authenticate them.
```

Reply:
This is the exactly point of the use case, that the FW cannot be measured before execution..

Reply:
omitting this suggestion as per https://github.com/ietf-scitt/draft-birkholz-scitt-software-supply-chain-use-cases/pull/17#issuecomment-1430206115